vault-core/src/main/java/org/apache/jackrabbit/vault/fs/impl/io/DocViewImporter.java

```diff
@@ -110,23 +111,30 @@
     static final Logger log = LoggerFactory.getLogger(DocViewImporter.class);
 
     /**
-     * these properties are protected but can be set nevertheless via system view xml import
+     * these properties are protected but are set for new nodes nevertheless via system view xml import
      */
-    static final Set<Name> PROTECTED_PROPERTIES;
+    static final Set<Name> PROTECTED_PROPERTIES_CONSIDERED_FOR_NEW_NODES;
+
+    /**
+     * these properties are protected but are set for updated nodes via special JCR methods
+     */
+    static final Set<Name> PROTECTED_PROPERTIES_CONSIDERED_FOR_UPDATED_NODES;
 
     static {
         Set<Name> props = new HashSet<>();
         props.add(NameConstants.JCR_PRIMARYTYPE);
         props.add(NameConstants.JCR_MIXINTYPES);
         props.add(NameConstants.JCR_UUID);
+        PROTECTED_PROPERTIES_CONSIDERED_FOR_UPDATED_NODES = Collections.unmodifiableSet(props);
         props.add(NameConstants.JCR_ISCHECKEDOUT);
         props.add(NameConstants.JCR_BASEVERSION);
         props.add(NameConstants.JCR_PREDECESSORS);
         props.add(NameConstants.JCR_SUCCESSORS);
         props.add(NameConstants.JCR_VERSIONHISTORY);
```

Comment:
@tripodsan Any idea why these properties are imported via sysview for new nodes but
a) never exported through DocViewSaxFormatter
b) never imported for an update